@ndlib/gatsby-theme-marble/src/components/Shared/SearchTools/SearchResults/HitDisplay/test.js

```diff
@@ -24,3 +24,16 @@ test('HitDisplay', () => {
   expect(wrapper.find(DisplayViewToggle).exists()).toBeTruthy()
   expect(wrapper.find(ManifestCard).length).toEqual(3)
 })
+
+test('all metadata hightlights', () => {
+  const wrapper = shallow(<HitDisplay hits={hits} />)
+  expect(wrapper.find(DisplayViewToggle).exists()).toBeTruthy()
+  expect(wrapper.find(ManifestCard).at(0).children().html()).toEqual('<div class="css-1itje8o"><em>data</em></div>')
+  expect(wrapper.find(ManifestCard).at(2).children().html()).toEqual('<div class="css-1itje8o"><em>line2data</em></div>')
+})
+
+test('all metadata hightlights splits on :: and shows only the line with the data', () => {
+  const wrapper = shallow(<HitDisplay hits={hits} />)
+  expect(wrapper.find(DisplayViewToggle).exists()).toBeTruthy()
+  expect(wrapper.find(ManifestCard).at(2).children().html()).toEqual('<div class="css-1itje8o"><em>line2data</em></div>')
+})
```

Comment:
Do we need an additional test when there are no hits (default empty search scenario)?

Reply:
yes